package.json

```diff
@@ -31,10 +31,12 @@
   "devDependencies": {
     "@auto-it/conventional-commits": "^11.0.4",
     "@auto-it/npm": "^11.0.4",
+    "@types/fs-extra": "^11.0.4",
     "auto": "^11.0.4",
     "expo-module-scripts": "^3.4.1",
     "expo-modules-core": "^1.11.8",
-    "prettier": "^3.0.3"
+    "fs-extra": "^9.0.0",
```

Comment:
The implementation is inspired by the `withAndroidIcons` function of the `prebuild-config` expo package.
We can try to re-implement with only `fs` functions, but it may be more error-prone.

Reply:
I see! I can quickly try to remove this dependency and I will let you know.

Reply:
@pchalupa done, please take a look and try